python/lsst/cp/verify/utils.py

```diff
@@ -18,27 +18,261 @@
+    74: (6, 2),
+    75: (7, 0),
+    76: (7, 1),
+    77: (7, 2),
+    78: (8, 0),
+    79: (8, 1),
+    80: (8, 2),
+    81: (6, 3),  # R21
+    82: (6, 4),
+    83: (6, 5),
+    84: (7, 3),
+    85: (7, 4),
+    86: (7, 5),
+    87: (8, 3),
+    88: (8, 4),
+    89: (8, 5),
+    90: (6, 6),  # R22
+    91: (6, 7),
+    92: (6, 8),
+    93: (7, 6),
+    94: (7, 7),
+    95: (7, 8),
+    96: (8, 6),
+    97: (8, 7),
+    98: (8, 8),
+    99: (6, 9),  # R23
+    100: (6, 10),
+    101: (6, 11),
+    102: (7, 9),
+    103: (7, 10),
+    104: (7, 11),
+    105: (8, 9),
+    106: (8, 10),
+    107: (8, 11),
+    108: (6, 12),  # R24
+    109: (6, 13),
+    110: (6, 14),
+    111: (7, 12),
+    112: (7, 13),
+    113: (7, 14),
+    114: (8, 12),
+    115: (8, 13),
+    116: (8, 14),
+    117: (9, 0),  # R30
+    118: (9, 1),
+    119: (9, 2),
+    120: (10, 0),
+    121: (10, 1),
+    122: (10, 2),
+    123: (11, 0),
+    124: (11, 1),
+    125: (11, 2),
+    126: (9, 3),  # R31
+    127: (9, 4),
+    128: (9, 5),
+    129: (10, 3),
+    130: (10, 4),
+    131: (10, 5),
+    132: (11, 3),
+    133: (11, 4),
+    134: (11, 5),
+    135: (9, 6),  # R32
+    136: (9, 7),
+    137: (9, 8),
+    138: (10, 6),
+    139: (10, 7),
+    140: (10, 8),
+    141: (11, 6),
+    142: (11, 7),
+    143: (11, 8),
+    144: (9, 9),  # R33
+    145: (9, 10),
+    146: (9, 11),
+    147: (10, 9),
+    148: (10, 10),
+    149: (10, 11),
+    150: (11, 9),
+    151: (11, 10),
+    152: (11, 11),
+    153: (9, 12),  # R34
+    154: (9, 13),
+    155: (9, 14),
+    156: (10, 12),
+    157: (10, 13),
+    158: (10, 14),
+    159: (11, 12),
+    160: (11, 13),
+    161: (11, 14),
+    197: (12, 0),  # R40
+    198: (14, 2),
+    199: (12, 2),
+    200: (12, 1),
+    162: (12, 3),  # R41
+    163: (12, 4),
+    164: (12, 5),
+    165: (13, 3),
+    166: (13, 4),
+    167: (13, 5),
+    168: (14, 3),
+    169: (14, 4),
+    170: (14, 5),
+    171: (12, 6),  # R42
+    172: (12, 7),
+    173: (12, 8),
+    174: (13, 6),
+    175: (13, 7),
+    176: (13, 8),
+    177: (14, 6),
+    178: (14, 7),
+    179: (14, 8),
+    180: (12, 9),  # R43
+    181: (12, 10),
+    182: (12, 11),
+    183: (13, 9),
+    184: (13, 10),
+    185: (13, 11),
+    186: (14, 9),
+    187: (14, 10),
+    188: (14, 11),
+    201: (14, 12),  # R44
+    202: (12, 14),
+    203: (12, 12),
+    204: (13, 12),
+}
 
 
 def mergeStatDict(statDictA, statDictB):
     """Merge one dictionary of statistics with another.
 
     Parameters
     ----------
-    statDictA, statDictB : `dict` [`str`, `dict`]
+    statDictA, statDictB: `dict` [`str`, `dict`]
         Input dictionaries of statistics.
 
     Returns
     -------
-    output : `dict` [`str`, `dict`]
+    output: `dict` [`str`, `dict`]
         A dictionary containing the union of the internal
         dictionaries of ``statDictA`` and ``statDictB``, indexed
         by the amplifier names.
 
     Raises
     ------
-    RuntimeError :
+    RuntimeError:
```

Comment:
But the dev guide also suggests this shouldn't have a `:` at all, so I'm learning new things too: https://developer.lsst.io/python/numpydoc.html#parameters